tools/sklearn/keras_image_train_and_eval.xml

```diff
@@ -0,0 +1,281 @@
+<tool id="keras_image_train_and_eval" name="Image Deep Learning" version="@VERSION@">
+    <description>performs training and evaluation using image data generator</description>
+    <macros>
+        <import>main_macros.xml</import>
+        <import>keras_macros.xml</import>
+    </macros>
+    <expand macro="python_requirements"/>
+    <expand macro="macro_stdio"/>
+    <version_command>echo "@VERSION@"</version_command>
+    <command>
+        <![CDATA[
+        export HDF5_USE_FILE_LOCKING='FALSE';
+        #for $f in $input_options.infile_images#
+            unzip -o -q '$f' -d "`pwd`/images";
+        #end for
+        python '$__tool_directory__/keras_image_train_and_eval.py'
+            --inputs '$inputs'
+            --estimator '$experiment_schemes.infile_estimator'
+            --infile_images '$input_options.infile_images'
+            --infile_dataframe '$input_options.infile_dataframe'
+            --outfile_result "`pwd`/tmp_outfile_result"
```

Comment:
here also `./`

Reply:
Thanks a lot for all those suggestions. However, full path is needed here, and probably in all python scripts. Lost track of relative paths might happen on clusters.